internal/cli/events/orgs_list.go

```diff
@@ -105,7 +105,10 @@ func OrgListBuilder() *cobra.Command {
 		Aliases: []string{"ls"},
 		Args:    require.NoArgs,
 		Example: `  # Return a JSON-formatted list of events for the organization with the ID 5dd5a6b6f10fab1d71a58495:
-  atlas events organizations list --orgId 5dd5a6b6f10fab1d71a58495 --output json`,
+  atlas events organizations list --orgId 5dd5a6b6f10fab1d71a58495 --output json
+  
+  # Return a JSON-formatted list of events between 2024-03-18T14:40:03-0000 and 2024-03-18T15:00:03-0000 and for the project with the ID 5e2211c17a3e5a48f5497de3 bew
```

Comment:
is this part of the ID?
```suggestion
  # Return a JSON-formatted list of events between 2024-03-18T14:40:03-0000 and 2024-03-18T15:00:03-0000 and for the project with the ID 5e2211c17a3e5a48f5497de3
```